lib/db.js

```diff
@@ -51,15 +51,10 @@ function getDbAux(url, component, callback) {
     client.connect(
         url,
         {
-            db: {
-                bufferMaxEntries: config.checkDB.bufferMaxEntries,
-                domainsEnabled: true
-            },
-            server: {
-                reconnectTries: config.checkDB.reconnectTries,
-                reconnectInterval: config.checkDB.reconnectInterval,
-                domainsEnabled: true
-            }
+            bufferMaxEntries: config.checkDB.bufferMaxEntries,
```

Comment:
This modifications avoid the following warning in log traces:

```
the server/replset/mongos options are deprecated, all their options are supported at the top level of the options object [poolSize,ssl,sslValidate,sslCA,sslCert,ciphers,ecdhCurve,sslKey,sslPass,sslCRL,autoReconnect,noDelay,keepAlive,connectTimeoutMS,family,socketTimeoutMS,reconnectTries,reconnectInterval,ha,haInterval,replicaSet,secondaryAcceptableLatencyMS,acceptableLatencyMS,connectWithNoPrimary,authSource,w,wtimeout,j,forceServerObjectId,serializeFunctions,ignoreUndefined,raw,bufferMaxEntries,readPreference,pkFactory,promiseLibrary,readConcern,maxStalenessSeconds,loggerLevel,logger,promoteValues,promoteBuffers,promoteLongs,domainsEnabled,keepAliveInitialDelay,checkServerIdentity,validateOptions,appname,auth]
```

NTC (informative)